index.html

```diff
@@ -6,33 +6,80 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Bang</title>
-    <link rel="stylesheet" href="./resources/css/reset.css">
-    <link rel="stylesheet" href="./resources/css/style.css">
+    <link rel="stylesheet" href="resources/css/reset.css">
+    <link rel="stylesheet" href="resources/css/style.css">
 </head>
-
 <body>
     <div class="background">
         <div class="opacity">
             <div class="player-pair">
-                <div class="player">
+                <div class="player moved">
                     <div class="hp-bar">
                         <div class="hp"></div>
                         <div class="hp"></div>
                         <div class="hp"></div>
                         <div class="hp"></div>
                         <div class="hp"></div>
                     </div>
-                    <div class="card"></div>
+                    <div>
+                        <div class="portait-wrap">
+                            <div class="portait player-active"></div>
+                        </div>
+                        <div class="buff-list right">
+                            <div class="buff-wrap">
+                                <div class="buff buff-1"></div>
+                            </div>
+                            <div class="buff-wrap">
+                                <div class="buff buff-2"></div>
+                            </div>
+                            <div class="buff-wrap">
+                                <div class="buff buff-3"></div>
+                            </div>
+                            <div class="buff-wrap">
+                                <div class="buff buff-4"></div>
+                            </div>
+                            <div class="buff-wrap">
+                                <div class="buff buff-5"></div>
+                            </div>
+                        </div>
+                        <div class="weapon-wrap">
+                            <div class="weapon"></div>
+                        </div>
+                    </div>
                 </div>
-                <div class="player">
-                    <div class="hp-bar">
+                <div class="player moved">
+                    <div class="hp-bar hp-bar-right">
                         <div class="hp"></div>
                         <div class="hp"></div>
                         <div class="hp"></div>
                         <div class="hp"></div>
                         <div class="hp"></div>
                     </div>
-                    <div class="card"></div>
+                    <div>
+                        <div class="portait-wrap">
+                            <div class="portait portait-right"></div>
+                        </div>
+                        <div class="buff-list">
+                            <div class="buff-wrap">
+                                <div class="buff buff-reverse-1"></div>
```

Comment:
can you avoid hard numeration in classes? looks like we have only five buffs max, no more